Tests/BodegaTests/SQLiteStorageEngineTests.swift

```diff
@@ -3,13 +3,24 @@ import XCTest
 
 final class SQLiteStorageEngineTests: XCTestCase {
 
-    private var storage: SQLiteStorageEngine!
+    private static var _storage: SQLiteStorageEngine!
```

Comment:
Perhaps I'm not familiar with this pattern but is there a reason to create this new property (`_storage`) and then have a computed property below?

Reply:
Ha, yeah It probably would have benefitted to have added a comment here. Through a combination of necessity, and not wanting to make massive changes to this test case, I had to get a bit creative.

The backing `sqlite3` file wasn't being deleted between test runs which was an issue during development, but to fix this requires that I hold onto a reference until the `static tearDown` method is called at the end of the test case. One of the delights of the SQLite C API is that the database connection pointer is needed to properly close the connection before deleting the backing file or else some API misuse warnings are thrown in the console.

So I'm essentially just creating a very rough singleton here in order to capture the reference to the store so that I can call it's internal delete command correctly while also keeping the rest of the test case from needing to reference `Self` every time it interacts with the store.